### DISCUSSION TITLE
Model viewer editior upload glitch

### DISCUSSION THREAD

**Discussion by 1kaiser** [google/model-viewer#2438] [Q&A]
![Screenshot_20210531-221939_Pixel_Launcher](https://user-images.githubusercontent.com/26379748/120223016-8809f380-c25e-11eb-9b83-280a967291a0.png)
The upload button doesn't works while selecting objects from file system

**Comment by shamsail** [comment]
Can you check it again because it's working .. Did you try your model by drag and drop also?

**Reply by 1kaiser** [reply]
How can we drag and drop using android mobile?

**Comment by elalish** [comment]
Yeah, that appears to be true on Android. The editor isn't designed for mobile use. You can open the model on your desktop browser though, and then click Deploy Mobile in order to test it on Android/iOS using our QR code.

**Reply by 1kaiser** [reply]
The old editor used to have the uploading feature, but in the new editor the glb uploader doesn't work, even that should have solved
